runtime/src/iree/hal/drivers/vulkan/vulkan_headers.h

```diff
@@ -41,10 +41,10 @@
 #define VK_USE_PLATFORM_WIN32_KHR 1
 #endif  // IREE_PLATFORM_*
 
-#include <vulkan/vulkan.h>  // IWYU pragma: export
+#include "third_party/vulkan_headers/include/vulkan/vulkan.h"  // IWYU pragma: export
```

Comment:
Need to fix Bazel
https://github.com/openxla/iree/actions/runs/4853312751/jobs/8649339421?pr=13346#step:4:527

```
runtime/src/iree/hal/drivers/vulkan/vulkan_headers.h:44:10: fatal error: 'third_party/vulkan_headers/include/vulkan/vulkan.h' file not found
#include "third_party/vulkan_headers/include/vulkan/vulkan.h"  // IWYU pragma: export
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
```

https://github.com/openxla/iree/blob/main/build_tools/third_party/vulkan_headers/BUILD.overlay